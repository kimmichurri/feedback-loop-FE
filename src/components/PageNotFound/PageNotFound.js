import React from 'react';

export const PageNotFound = () => {
	return (
		<div className='page-not-found'>
			<h2 className='not-found-text'>This is not the page</h2>
			<h2 className='not-found-text'>you are looking for.</h2>
			<h2 className='not-found-text'>404</h2>
		</div>
	)
}

export default PageNotFound