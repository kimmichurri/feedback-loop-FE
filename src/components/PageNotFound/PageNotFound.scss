.page-not-found {
 margin-top: 10%;
}

.not-found-text {
  font-family: $main-font;
  color: #FFF;
  font-size: 4.5em;
}