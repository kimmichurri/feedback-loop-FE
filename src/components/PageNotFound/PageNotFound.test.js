import React from 'react'
import { PageNotFound } from './PageNotFound'
import { shallow } from 'enzyme'

describe('PageNotFound', () => {
  let wrapper
  it('should match the snapshot', () => {
    wrapper = shallow(<PageNotFound />)
    expect(wrapper).toMatchSnapshot()
  })
})