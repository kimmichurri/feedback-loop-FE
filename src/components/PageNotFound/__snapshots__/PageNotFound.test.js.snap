// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match the snapshot 1`] = `
<div
  className="page-not-found"
>
  <h2>
    This is not the page
  </h2>
  <h2>
    you are looking for.
  </h2>
  <h2
    className="not-found"
  >
    404
  </h2>
</div>
`;
